build.sh

```diff
@@ -35,9 +35,9 @@ as_root ln -s /usr/lib64/libbpf.so.1 /usr/lib/x86_64-linux-gnu/libbpf.so.1 2>/de
 as_root ldconfig
 
 # Run unit tests
-export LD_LIBRARY_PATH="${PREFIX_DIR}/usr/local/lib:/usr/local/lib64"
-"${MCM_BUILD_DIR}/bin/sdk_unit_tests"
-"${MCM_BUILD_DIR}/bin/media_proxy_unit_tests"
+export LD_LIBRARY_PATH="${PREFIX_DIR}/usr/local/lib:/usr/local/lib"
+ctest --output-on-failure --test-dir "${MCM_BUILD_DIR}" -V
+
```

Comment:
Please verify what you're changing here

Reply:
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/286/commits/316ac7128e8def77fd3380b6459edd0884ea023c